core/unit_test/TestMDRange.hpp

```diff
@@ -2187,6 +2264,7 @@ struct TestMDRange_5D {
 
       TestMDRange_5D::HostViewType h_view = Kokkos::create_mirror_view( v );
       Kokkos::deep_copy( h_view, v );
+      Kokkos::fence();
```

Comment:
deep_copy without a execution space instance argument is synchronous with respect to the calling thread. deep_copy(HPX(),h_view,v); would be asynchronous with respect to the calling thread (but will fence against any other kernel launched on HPX).

Reply:
Thanks for pointing that out. I'll remove those fences.